app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -24,6 +24,7 @@ class PipelineSampleReport extends React.Component {
       class: "prog-bar"
     });
     this.admin = props.admin;
+    this.allowedFeatures = props.allowedFeatures;
```

Comment:
maybe you can also set another object variable 
this.allowPhyloTree = this.allowedFeatures.indexOf("phylo_trees") >= 0  
and use it as follows. This will make it slightly easier if you have other way to check it in the future. 

Also, do you have to check if this project is readonly or not for the user?

Reply:
Thanks, changed both